shop/src/main/java/ro/msg/learning/shop/Mapper/ProductCategorySimpleTranslatorMapper.java

```diff
@@ -0,0 +1,24 @@
+package ro.msg.learning.shop.Mapper;
+
+import org.springframework.stereotype.Component;
+import ro.msg.learning.shop.DTO.ProductCategorySimpleDto;
+import ro.msg.learning.shop.Domain.ProductCategory;
+
+@Component
+public class ProductCategorySimpleTranslatorMapper {
+    public ProductCategorySimpleDto toProductCategorySimpleDto(ProductCategory productCategory) {
+        ProductCategorySimpleDto productCategorySimpleDto= new ProductCategorySimpleDto();
+        productCategorySimpleDto.setId(productCategory.getId());
+        productCategorySimpleDto.setName(productCategory.getName());
+        productCategorySimpleDto.setDescription(productCategory.getDescription());
+        return productCategorySimpleDto;
+    }
+
+    public ProductCategory toProductCategory(ProductCategorySimpleDto productCategorySimpleDto) {
+        ProductCategory productCategory=new ProductCategory();
```

Comment:
You might want to check `@Builder` from lombok. We can use it to create objects in a more elegant way.
Reference: https://www.baeldung.com/lombok-builder